pandas/api/io/__init__.py

```diff
@@ -0,0 +1,2 @@
+""" Public IO API """
+from pandas.formats.style import Styler  # noqa
```

Comment:
I think it should be just be `pandas.io`?  The second level `api` are `import *` into the top level.